Converted ARCore samples to use Sceneform for rendering.

claywilkinson opened the pull request:
This is the most direct conversion from using OpenGL to using
Sceneform for the Augmented Images sample.  Only the rendering has been changed.

A more dramatic change is done for the Cloud Anchors sample, which
 extends ArFragment and configure the
ARCore Session by overriding getSessionConfiguration() to enable
cloud anchors. This  eliminates the camera permission,
 and session management boilerplate.

claywilkinson committed:
Converted Augmented Image sample to use Sceneform's ARSceneView

This is the most direct conversion from using OpenGL to using
Sceneform.  Only the rendering has been changed.

Another approach would be to extend ArFragment and configure the
ARCore Session by overriding getSessionConfiguration() to enable
augmented images.  This would then elminate the camera permission,
full screen handling, and session management boilerplate.

claywilkinson committed:
Merge pull request #3 from claywilkinson/master

merging upstream changes into sceneform branch.

claywilkinson committed:
Converting Cloud Anchors Java sample to Sceneform.

claywilkinson committed:
Updating sample to add boxes as child objects.

This demonstrates how to share the local position relative
to the anchor with multiple objects.

claywilkinson committed:
Updating PointCloudNode to draw pyramids per feature.

claywilkinson committed:
Fixing point cloud vertex positions.

rohitagarwal3011 commented:
This is written in google's documentation of [Working with cloud anchors](https://developers.google.com/ar/develop/developer-guides/anchors)

**Keep objects close to anchors**
When anchoring objects, make sure that they are close to the anchor you are using. Avoid placing objects farther than a few meters from the anchor to prevent unexpected rotational movement due to ARCore's updates to world space coordinates.

If you need to place an object more than a few meters away from an existing anchor, create a new anchor closer to this position and attach the object to the new anchor.

My question is that , if we host multiple cloud anchors for objects that are placed few meters away, then while we are resolving them, how can we resolve all the cloud-anchors by scanning at a particular place?

claywilkinson committed:
Adding Sceneform implementation of computervision_java sample.

This moves the cpu camera image into a bitmap that is rotated correctly.

The GPU image capture currently is not supported in Sceneform since
there is no access to OpenGL context via sceneform.
